nion/instrumentation/MultiAcquire.py

```diff
@@ -217,89 +243,66 @@ def __set_beam_blanker(self, blanker_on):
-        return crop_ranges
-
-    def stitch_spectra(self, spectra, crop_ranges):
-        result = numpy.zeros(crop_ranges[-1][2][1])
-        last_mean = None
-        last_overlap = None
-        for i in range(len(crop_ranges)):
-            data = (spectra[i].data)/self.__active_spectrum_parameters[i]['exposure_ms']
-            # check if hdr needs to be done in this interval, which is true when the overlap with the next one is 100%
-            if i > 0 and (crop_ranges[i-1][1][1] - crop_ranges[i-1][1][0]) == 0:
-                data[data>self.__active_settings['saturation_value']] = (spectra[i-1].data[[data>self.__active_settings['saturation_value']]])/self.__active_spectrum_parameters[i]['exposure_ms']
-
-            data = numpy.sum(data[crop_ranges[i][0][0]:crop_ranges[i][0][1]], axis=0)
-            if last_mean is not None:
-                data -= numpy.mean(data[crop_ranges[i][1][0]:last_overlap]) - last_mean
-                print(last_mean, last_overlap)
-            if self.__active_settings['y_align'] and i < len(crop_ranges) - 1:
-                last_overlap = data.shape[-1] - crop_ranges[i][1][1]
-                last_mean = numpy.mean(data[crop_ranges[i][1][1]:])
-            result[crop_ranges[i][2][0]:crop_ranges[i][2][1]] = data[crop_ranges[i][1][0]:crop_ranges[i][1][1]]
-
-        return result
+        self.set_progress_counter(0, 0, 100)
 
     def __acquire_multi_acquire_data(self, number_pixels, line_number=0, flyback_pixels=0):
         for parameters in self.__active_spectrum_parameters:
-            print('start preparations')
+            logging.debug('start preparations')
             starttime = time.time()
             if self.abort_event.is_set():
                 break
             self.shift_x(parameters['offset_x'])
-            self.shift_y(parameters['offset_y'])
             self.adjust_focus(parameters['offset_x'])
             frame_parameters = self.camera.get_current_frame_parameters()
             frame_parameters['exposure_ms'] =  parameters['exposure_ms']
             frame_parameters['processing'] = 'sum_project' if self.__active_settings['bin_spectra'] else None
             self.camera.set_current_frame_parameters(frame_parameters)
             self.camera.acquire_sequence_prepare(parameters['frames']*number_pixels+flyback_pixels)
-            print('finished preparations in {:g} s'.format(time.time() - starttime))
+            logging.debug('finished preparations in {:g} s'.format(time.time() - starttime))
             starttime = 0
-            print('start sequence')
+            logging.debug('start sequence')
             starttime = time.time()
             data_element = self.camera.acquire_sequence(parameters['frames']*number_pixels+flyback_pixels)
             if data_element:
                 data_element = data_element[0]
-            print('end sequence in {:g} s'.format(time.time() - starttime))
+            logging.info('end sequence in {:g} s'.format(time.time() - starttime))
```

Comment:
Should this be `info` or `debug`?

Reply:
I left some messages on level "info" on purpose, but thinking about it now, I think you're right an it is all debug output. So all logging messages are level "debug" now.